repository-data/webfiles/src/main/resources/site/freemarker/common/macro/sections/tableauLookupSection.ftl

```diff
@@ -78,6 +79,9 @@
             var viz${index}LoadingRetryAtempIntervales;
             var viz${index}LoadingTimer;
 
+            var downloadVizLink;
+            var showPostcode;
```

Comment:
These variables need to be preceded with 'viz${index}' as there can be more than one instance of Tableau per page. Also see note on line 166 about removing 'downloadVizLink' from the global scope.